web/src/installers/installer-versions.ts

```diff
@@ -41,3 +68,25 @@ export async function getInstallerVersions(distUrl: string, kurlVersion?: string
   installerVersionsCache[url] = installerVersions;
   return installerVersionsCache[url];
 }
+
+async function externalAddonHandler() {
+  try {
+    const response = await fetch("https://kurl-sh.s3.amazonaws.com/external/addon-registry.json");
```

Comment:
Is this URL something we want to make configurable in case it needs to be changed in the future for some reason?

Reply:
I thought about it, but I figured we can cross that bridge if we ever come to it